docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,9 @@
+= {project-name} Documentation
```

Comment:
It's too verbose, I think we could go for "Introduction" or "Browser Extension Documentation"

Reply:
I agree that in general, terse is better, especially in the TOC and within.